### ISSUE TITLE
Clean up from Xcode 10.2 changes

### ISSUE THREAD

**Issue by ktkopone** [snipsco/Postal#84]
As mentioned in the recent XCode 10.2 PR thread: https://github.com/snipsco/Postal/pull/83#issuecomment-485564820

* Result module was removed but is still referenced in Postal.podspec. I don't use pods in my project so I'm not sure what issues this may cause.

* Postal-iOS framework has 7 swift language-related warnings:
<img width="741" alt="Screen Shot 2019-04-22 at 5 39 00 PM" src="https://user-images.githubusercontent.com/26582629/56534036-dcdacc00-6526-11e9-8e15-db4caf181d59.png">

**Comment by klefevre** [comment]
This should be fixed by #85. Don't hesitate to give a try. I'll wait for feedback before releasing a new version. Thanks

**Comment by ktkopone** [comment]
Looks mostly good, noticed one more warning upon re-opening XCode about an english language setting being deprecated:

<img width="418" alt="Screen Shot 2019-05-07 at 1 18 58 PM" src="https://user-images.githubusercontent.com/26582629/57319511-b5f5cb80-70ca-11e9-8a1c-a302d7184219.png">

the prompted auto-fix did the following and seemed to resolve it:

```
diff --git a/Postal.xcodeproj/project.pbxproj b/Postal.xcodeproj/project.pbxproj

index c06c1f9..3eed3be 100644
--- a/Postal.xcodeproj/project.pbxproj
+++ b/Postal.xcodeproj/project.pbxproj
@@ -584,10 +584,9 @@
                        };
                        buildConfigurationList = 98DB008F1CEC6983003CAABB /* Build configuration list for PBXProject "Postal" */;
                        compatibilityVersion = "Xcode 3.2";
-                       developmentRegion = English;
+                       developmentRegion = en;
                        hasScannedForEncodings = 0;
                        knownRegions = (
-                               English,
                                en,
                                Base,
                        );
```
